src/components/header/navMenu.js

```diff
@@ -1,37 +1,56 @@
 import React from 'react';
-import { NavLink } from 'react-router-dom';
+import { NavLink, withRouter } from 'react-router-dom';
 import PropTypes from 'prop-types';
 
 import styles from './navMenu.module.scss';
 
-const NavMenu = ({ selectedTeam: { teamColor, urlName }, playerId }) => {
-  const activeStyle = {
-    color: teamColor,
+const NavMenu = ({
+  history: {
+    location: { pathname },
+  },
+  selectedTeam: { teamColor, urlName },
+  playerId,
+}) => {
+  const activeBorder = {
+    position: 'absolute',
+    width: 'calc(100% - 40px)',
+    left: '20px',
+    bottom: '0',
```

Comment:
Add these css variables as a class and no need for absolute, better like so:

```
.navLink {
  padding-top: 7px;
  border-bottom: 4px solid transparent;
}
```

example:

```
<div className={styles.navLink} style={getActiveStyle('/standings')} />
```